sample-compose/app/build.gradle.kts

```diff
@@ -125,6 +125,7 @@ dependencies {
 
     implementation("androidx.activity:activity-ktx:${Versions.ANDROIDX_ACTIVITY_KTX_VERSION}")
     implementation("androidx.appcompat:appcompat:${Versions.ANDROIDX_SUPPORT_VERSION}")
+    implementation("androidx.constraintlayout:constraintlayout:${Versions.CONSTRAINT_LAYOUT_VERSION}")
```

Comment:
Do we still need this library? We can probably replace the `ConstraintLayout` in `activity_main.xml` with a `LinearLayout` or something, what do you think? 😄